firebase-config/src/test/java/com/google/firebase/remoteconfig/FirebaseRemoteConfigTest.java

```diff
@@ -334,6 +336,8 @@ public void onError(@NonNull FirebaseRemoteConfigException error) {
             listeners,
             mockRetryListener,
             scheduledExecutorService);
+    realtimeMetadataClient =
```

Comment:
Are there other tests we can add to assert the backoff behavior?

Reply:
Added some more tests here that go through the path of Realtime